site/src/scaffold/token.html

```diff
@@ -10,37 +10,50 @@
 		<meta name="color-scheme" content="light dark" />
 		<link rel="stylesheet" href="~/src/scaffold/index.scss"
 	</head>
-	<body>
-		<header class="padding-block-xs padding-inline-s font-weight-bold" style="border-bottom: 1px solid var(--theme-border)">
-			Atlas Design
+	<body id="body">
+		<header id="header" class="padding-block-xs padding-inline-s font-weight-bold" style="border-bottom: 1px solid var(--theme-border)">
+			<a href="~/src/index.md" class="color-text text-decoration-none font-weight-bold">Atlas Design</a>
 		</header>
-		<main class="padding-s">
-			<div class="markdown">
+		<main id="main">
+			<div class="layout-container padding-top-s">
 				{{{ body }}}
-			</div>
-			
-			<div class="padding-block-m">
-				<a href="https://github.com/microsoft/atlas-design/tree/main{{cssTokenSource}}">View on Github</a>
-			</div>
-			<pre><code>{{#tokens}}{{name}}: {{value}}
+				
+				<div class="padding-block-m">
+					<a href="https://github.com/microsoft/atlas-design/tree/main{{cssTokenSource}}">View on Github</a>
+				</div>
+				<div class="code-block">
+					<div class="code-block-body">
+					<pre><code>{{#tokens}}{{name}}: {{value}}
 {{/tokens}}</code></pre>
```

Comment:
Can you fix the indent here, please? Looks off a bit.

Reply:
I would like to , but this is a `<pre>` tag, so we need like this for now to ensure the code renders correctly.

Reply:
Oh, right! ok, resolving